src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/containerized-images.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Instrument your containerized image layer
+metaDescription: A guide for instrumenting your containerized image layer with New Relic.
+freshnessValidatedDate: never
+---
+
+If you're using a containerized image for Lambda, you'll want to add New Relic to your code. After you add our pre-built layer to your image, you'll need to push the image to your AWS Elastic Container Registry. From there, whenever you run your function, New Relic will run, too. 
+
+Here's a guide to adding our pre-built layer to your code. 
+
+<Steps>
+<Step>
+### Add New Relic to your Dockerfile:
+New Relic offers pre-built layers for different runtimes: Java, Node.js, and Python.
+
+1. Select the layer compatible with your specific Lambda function's runtime and region:
+ * [Java](https://gallery.ecr.aws/x6n7b2o2/newrelic-lambda-layers-java)
+ * [Node.js](https://gallery.ecr.aws/x6n7b2o2/newrelic-lambda-layers-nodejs) 
+ * [Python](https://gallery.ecr.aws/x6n7b2o2/newrelic-lambda-layers-python)
+3. **At the top of your Dockerfile** add the following code: 
+
+ <Tabs>
+ <TabsBar>
+ <TabsBarItem id="1">Dockerfile for Java</TabsBarItem>
+ <TabsBarItem id="2">Dockerfile for Node.js</TabsBarItem>
+ <TabsBarItem id="3">Dockerfile for Python</TabsBarItem>
+ </TabsBar>
+
+ <TabsPages>
+ <TabsPageItem id="1`">
+ ```
+ FROM public.ecr.aws/x6n7b2o2/newrelic-lambda-layers-java:17-arm64 AS layer
+
+ # Layer code
+ COPY --from=layer /opt/ /opt/
+ ```
+ </TabsPageItem>
+
+ <TabsPageItem id="2`">
+ ```
+ FROM public.ecr.aws/x6n7b2o2/newrelic-lambda-layers-python:38-arm64 AS layer
+
+ # Layer code
+ COPY --from=layer /opt/ /opt/
+ ```
+ </TabsPageItem>
+
+ <TabsPageItem id="3`">
+ ```
+ FROM public.ecr.aws/x6n7b2o2/newrelic-lambda-layers-nodejs:20-arm64 AS layer
+
+ # Layer code
+ COPY --from=layer /opt/ /opt/
+ ```
+ </TabsPageItem>
+ </TabsPages>
+</Tabs>
+</Step>
+
+<Step>
+### Create an image
+ 1. After you add the New Relic pre-built layer to your Dockerfile, you will need to build a new image from the Dockerfile. This new image will have your original code _and_ New Relic's layer. To do this, we recommend following the steps listed in the AWS guide, [Pushing a Docker image](https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-push-ecr-image.html).
+ 2. Push the new image to your AWS Elastic Container Registry (ECR) repository for deployment.
+</Step>
+
+<Step>
+### Configure your Lambda function
+ 1. Create a new Lambda function using the image you pushed to AWS ECR. For more information on how to create this function, we recommend reviewing the AWS guide: [Working with container images](https://docs.aws.amazon.com/lambda/latest/dg/images-create.html).
+ 2. Configure the Lambda function with the same settings as your original function, but ensure you add any required environment variables for New Relic integration.
```

Comment:
integration -> Lambda monitoring

Reply:
Sorry, I think I am a little confused about this comment. What should I change here?